### ISSUE TITLE
MrBean support on Android

### ISSUE THREAD

**Issue by NebuPookins** [FasterXML/jackson-modules-base#197]
While the core Jackson library works on Android, the MrBean module does not. This is because MrBean uses ASM to dynamically generate bytecode -- but the class file used by the Dalvik VM can only read class files in the DEX format.

One possibility would be to use a library like https://github.com/linkedin/dexmaker

**Comment by cowtowncoder** [comment]
Yes, correct. MrBean as it is does not and cannot work on Android.

To support Android we would realistically need separate library, I think.

But I also do not know if it is possible to dynamically load generated byte (dex) code, even if generation was implemented.
